frontend/lib/profile/view/profile.view.dart

```diff
@@ -75,20 +76,12 @@ class _ProfileState extends State<Profile> {
   }
 
   Widget _buildListingsGrid(List<ListingItem> listings) {
-    final gridDelegate = const SliverGridDelegateWithFixedCrossAxisCount(
-      crossAxisCount: 2,
-      childAspectRatio: 0.75,
-      crossAxisSpacing: 8,
-      mainAxisSpacing: 8,
-      mainAxisExtent: 250,
-    );
-
     return Padding(
       padding: const EdgeInsets.all(2.0),
```

Comment:
We might want to keep consistent with the home page in terms of side padding. Right now, the grid view in the profile page seems to close to the screen boundaries. (Let me know if you and Marwan had other considerations for this).
<img width="428" alt="image" src="https://github.com/user-attachments/assets/6c585677-e65a-4251-9cbe-4358004f654f" />